src/components/ContentPreview/ContentPreview.js

```diff
@@ -196,13 +195,19 @@ class ContentPreview extends PureComponent<Props, State> {
     }
 
     /**
-     * Called after shell re-mounts
+     * After component updates, check if file ID or token has changed and if so, load Preview again.
      *
      * @private
      * @return {void}
      */
-    componentDidUpdate(): void {
-        this.loadPreview();
+    componentDidUpdate(prevProps: Props): void {
+        const { fileId, token }: Props = this.props;
```

Comment:
Why not just check it this.preview exists?
Because componentWillRecvProps will just nuke it with the same conditions as below?

Reply:
Ah this is annoying. You're right based on this code I can just check whether or not this.preview exists, but that reminds me that Instant Preview depends on the file being cached in the preview library's cache. Just means I need to either need to warm up not only ContentPreview's cache but also the preview library's cache OR on hover, call `updateFileCache` on that single file before calling `prefetch()`...